deps/rabbit/src/rabbit_connection_tracking.erl

```diff
@@ -68,10 +62,6 @@
 %% Sets up and resets connection tracking tables for this
 %% node.
 boot() ->
-    ensure_tracked_connections_table_for_this_node(),
-    ensure_per_vhost_tracked_connections_table_for_this_node(),
-    ensure_per_user_tracked_connections_table_for_this_node(),
-    clear_tracking_tables(),
     ok.
```

Comment:
It looks like we should also delete the following `rabbit` boot steps:
https://github.com/rabbitmq/rabbitmq-server/blob/56e4ed5464b3f23b682639378c4cab58879c288a/deps/rabbit/src/rabbit.erl#L212-L220

and delete functions `rabbit_connection_tracking:boot/0` and `rabbit_channel_tracking:boot/0` and the callback definition in https://github.com/rabbitmq/rabbitmq-server/blob/56e4ed5464b3f23b682639378c4cab58879c288a/deps/rabbit/src/rabbit_tracking.erl#L16